tests/testthat/test-run.R

```diff
@@ -18,21 +18,25 @@ test_that("Runs work", {
 
   model_summary <- summary(model)
 
-  r2 <- model_summary$r.squared
+  R2 <- model_summary$r.squared
   f <- model_summary$fstatistic[["value"]]
 
-  log_metric(
-    key = "R2",
-    value = r2
+  log_metrics(
+    R2,
+    "F" = f
   )
 
-  Sys.sleep(3)
-
-  log_metric(
-    "F",
-    f
+  expect_error(
```

Comment:
new test to check that `log_metrics` will fail if the size of `timestamp` (or `step`) is not compatible with the number of metrics passed to `...`